project_parent/views/project_parent.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+© 2017-2019 Elico Corp (https://www.elico-corp.com).
+License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+<odoo>
+    <record id="view_project_parent_edit_project" model="ir.ui.view">
+        <field name="name">project.project.form</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <field name="parent_project_id"/>
```

Comment:
this will not work.  unfortunately in v10 project had extension inheritance with account.analytic.account

 class Project(models.Model):

    inherits = {'account.analytic.account': "analytic_acount_id"}


this is not so anymore. So the field  parent_project is no more in analytic_account.
Being this connection gone. I suggest to replace this with parent_id. wich simply points to another project

so this requires a radical rethink ( and a migration script!)

Reply:
this will not work. unfortunately in v10 project had extension inheritance with account.analytic.account

class Project(models.Model):

inherits = {'account.analytic.account': "analytic_acount_id"}
this is not so anymore. So the field parent_project is no more in analytic_account.
Being this connection gone. I suggest to replace this with parent_id. wich simply points to another project

so this requires a radical rethink ( and a migration script!)


The solution would be to rewrite the thing with a simple project hierarchy, using odoo hierarchical relations, management:
 https://github.com/OCA/OCB/blob/12.0/odoo/models.py#L2262

And you  can also take care of writing a migration script to get the info in the previous datastructure and restore it.